index.js

```diff
@@ -54,7 +69,18 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies){
+  let sumMov = 0;
+ let aveMov = 0;
+  for(let i = 0; i < movies.length; i++){
+    let sumImdb = Number(movies[i].imdbRating)
+    sumMov += sumImdb
+    aveMov = sumMov/movies.length
+  }
+ return aveMov
```

Comment:
Same here - I think `average` and `sum` are expressive enough names for our variables.

That being said - currently we are dividing on line 78 every single time we loop.  Thats a lot of extra division that we really need to do only once.
We could loop through and then return `sumMov / movies.length`